### ISSUE TITLE
Bad link for k3d on windows

### ISSUE THREAD

**Issue by jlogelin-dell** [alexellis/arkade#589]
The reference is missing `.exe` from the download link `https://github.com/rancher/k3d/releases/download/v5.1.0/k3d-windows-amd64`

```
arkade get k3d
Downloading: k3d
2021/11/25 19:06:49 Looking up version for k3d
2021/11/25 19:06:49 Found: v5.1.0
Downloading: https://github.com/rancher/k3d/releases/download/v5.1.0/k3d-windows-amd64
Error: incorrect status for downloading tool: 404
```

**Comment by jlogelin** [comment]
PR: https://github.com/alexellis/arkade/pull/590

**Comment by alexellis** [comment]
Thanks @jlogelin please use "Fixes: #589" in the future, which was part of the PR template that you deleted :confused: 

Likewise, please fill out the issue templates otherwise it sets a poor example for others if we let this slide. You may feel like in this instance it didn't add anything, but for many issues it does - things that the submitter hasn't considered.
